config/webpack.config.js

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
Let's discuss the need for this. If the entry point is `foam-bin` and the only other thing we do is fix script tags in an html file, I think I'd prefer to not use `webpack`. It's bundling code that's already bundled by FOAM. We can just as easily copy `foam-bin` to `static/bundle` and refer to the correct location from `html` file(s).